<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qlspWindow</class>
 <widget class="QMainWindow" name="qlspWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(210, 210, 210,1);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tb_qlsp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sản phẩm</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Đơn vị sản xuất</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Số lượng</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Đơn giá</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tổng</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ngày nhập</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="tim_qlsp">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>50</y>
      <width>511</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
    </property>
    <property name="placeholderText">
     <string>Tìm kiếm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_qlsp">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Quản lý sản phẩm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="grb_qlch">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>270</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(210, 210, 210,1);</string>
    </property>
    <property name="title">
     <string>Thông tin</string>
    </property>
    <widget class="QLineEdit" name="txt_qlsp_sp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Tên sản phẩm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlsp_nsx">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Nơi sản xuất</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlsp_sl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Số lượng</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlsp_dg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Đơn giá</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlsp_sp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sản phẩm</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlsp_nsx">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nơi sản xuẩt</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlsp_sl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Số lượng</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlsp_dg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Đơn giá</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlsp_nn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ngày nhập</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QDateEdit" name="time_qlsp_nn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::DaySection</enum>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="date">
      <date>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlsp_tong">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Thành tiền</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlsp_dg_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tổng</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_timkiem_qlsp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
    </property>
    <property name="text">
     <string>Tìm kiếm</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn_again">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_xoa_qlsp">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(215, 220, 88, 0.77);</string>
     </property>
     <property name="text">
      <string>Xoá</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_thoat_qlsp">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(225, 130, 146, 1);</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sua_qlsp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_them_qlsp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
